src/Service.Tests/SqlTests/GraphQLPaginationTests/GraphQLPaginationTestBase.cs

```diff
@@ -932,6 +932,72 @@ public async Task TestColumnTieBreak()
             SqlTestHelper.PerformTestEqualJsonStrings(expected2, actual2.ToString());
         }
 
+        /// <summary>
+        /// Tests the GraphQL query for retrieving supported types with a specified number of rows.
```

Comment:
nit: need to update this description to note how this test is a 'best effort' attempt to reproduce an issue that occurs due to jsondocument disposal occurring before we are finished using it's values. As a result, this test attempts to catch circumstances where we don't call `JsonDocument.RootElement.Clone()`